src/test_l1_dsDisplay.c

```diff
@@ -78,28 +78,12 @@
 #include <ut_kvp_profile.h>
 #include "stdint.h"
 #include "dsDisplay.h"
-#include "dsVideoPortSettings.h"
+#include "test_parse_configuration.h"
```

Comment:
remove test_parse_configuration.h, if not used

Reply:
done